joomla/joomla-cms
# [com_users] all views: Add missing tooltips in search

#### Summary of Changes

Simple PR to add missing tootip to all views in the com_users search filters.
#### Testing Instructions

Code review.

or
1. Apply patch
2. Go to Users -> User Groups, check the tootip in the search field.
3. Go to Users -> Viewing Access Levels, check the tootip in the search field.
4. Go to Users -> User Notes, check the tootip in the search field.
5. Activate debug in global config
6. Go to Users -> Manage, click "Debug Permissions Report" and check the tootip in the search field.
7. Go to Users -> User Groups, click "Debug Permissions Report" and check the tootip in the search field.
